Source/WebKit/WebProcess/WebPage/WebPage.cpp

```diff
@@ -2901,12 +2901,12 @@ void WebPage::setSuppressScrollbarAnimations(bool suppressAnimations)
 {
     m_page->setShouldSuppressScrollbarAnimations(suppressAnimations);
 }
-    
```

Comment:
Please revert all these spacing changes and update your IDE configuration to not do this.

Reply:
Sorry about that--was trying out a different editor.